qa/workunits/rbd/rbd-nbd.sh

```diff
@@ -212,37 +220,54 @@ rbd bench ${IMAGE} --io-type write --io-size=1024 --io-total=1024
 
 # unmap by image name test
 DEV=`_sudo rbd device --device-type nbd map ${POOL}/${IMAGE}`
-get_pid
+get_pid ${POOL}
 unmap_device ${IMAGE} ${PID}
 DEV=
 
 # map/unmap snap test
 rbd snap create ${POOL}/${IMAGE}@snap
 DEV=`_sudo rbd device --device-type nbd map ${POOL}/${IMAGE}@snap`
-get_pid
+get_pid ${POOL}
 unmap_device "${IMAGE}@snap" ${PID}
 DEV=
 
 # map/unmap namespace test
 rbd snap create ${POOL}/${NS}/${IMAGE}@snap
 DEV=`_sudo rbd device --device-type nbd map ${POOL}/${NS}/${IMAGE}@snap`
-get_pid ${NS}
+get_pid ${POOL} ${NS}
 unmap_device "${POOL}/${NS}/${IMAGE}@snap" ${PID}
 DEV=
 
 # unmap by image name test 2
 DEV=`_sudo rbd device --device-type nbd map ${POOL}/${IMAGE}`
-get_pid
+get_pid ${POOL}
 pid=$PID
 DEV=`_sudo rbd device --device-type nbd map ${POOL}/${NS}/${IMAGE}`
-get_pid ${NS}
+get_pid ${POOL} ${NS}
 unmap_device ${POOL}/${NS}/${IMAGE} ${PID}
 DEV=
 unmap_device ${POOL}/${IMAGE} ${pid}
 
+# map/unmap test with just image name
+DEV=`_sudo rbd device --device-type nbd map ${IMAGE}`
```

Comment:
nit: theoretically one may change POOL variable not to match the default. I believe it was possible to do previously and the script would not still fail, but it will fail with your changes now.

Probably we should check that POOL matches the default (or just that its name is "rbd", which might not be correct for all cases but we could ignore) and skip the test if it is not the case?

Reply:
Yes, makes sense. Thanks